provider/aws/lambda/autoscale/handler.go

```diff
@@ -282,9 +282,9 @@ func desiredCapacity(largest, total *Metrics) (int64, error) {
 	// the number of instances over the amount required to fit the widest service (gen1 loadbalancer-facing)
 	extraWidth := totalCount - largest.Width
 
-	fmt.Printf("extraCapacity = %+v\n", extraCapacity)
-	fmt.Printf("extraFit = %+v\n", extraFit)
-	fmt.Printf("extraWidth = %+v\n", extraWidth)
+	log("extraCapacity = %+v\n", extraCapacity)
```

Comment:
seems like a lot of these should be `debug()` instead of `log()`

Reply:
Ah, I think I misunderstood from your previous comments.  I thought you intended to chain the functions together... I'll update so that most are calling debug directly.